packages/server/src/handler.js

```diff
@@ -11,13 +11,37 @@ import { access } from '@ucanto/validator'
  * @returns {API.ServiceMethod<API.Capability<A, R, C>, Exclude<U, {error:true}>, Exclude<U, Exclude<U, {error:true}>>>}
  */
 
-export const provide =
-  (capability, handler) =>
+export const provide = (capability, handler) =>
+  provideAdvanced({ capability, handler })
+
+/**
+ * @template {API.Ability} A
+ * @template {API.URI} R
+ * @template {API.Caveats} C
+ * @template {unknown} U
+ * @param {object} input
+ * @param {API.Reader<API.DID>} [input.audience]
+ * @param {API.CapabilityParser<API.Match<API.ParsedCapability<A, R, C>>>} input.capability
+ * @param {(input:API.ProviderInput<API.ParsedCapability<A, R, C>>) => API.Await<U>} input.handler
+ * @returns {API.ServiceMethod<API.Capability<A, R, C>, Exclude<U, {error:true}>, Exclude<U, Exclude<U, {error:true}>>>}
+ */
+
+export const provideAdvanced =
```

Comment:
Could there be a comment that would help me understand how to choose between `provide` and `provideAdvanced` e.g. by defining what 'advanced' means here?

Reply:
Added some code comments to address this